### PR TITLE
feat: add translations (zh-cn, zh-tw, jp)

### PR DESCRIPTION
**Pull request by hqod**
Hello,

Thank you for the excellent project. I have attached translations for simplified Chinese (zh-cn), traditional Chinese (zh-tw) and Japanese (jp) in this pull request for your review.

### CODE REVIEW

**Comment by RedstonePfalz** [comment]
I've checked your translations with a translator and they look good.

File: messages/jp.json [hunk 1]
```diff
@@ -144,7 +144,7 @@
 				"country": "国",
 				"phone": "電話番号",
 				"taxId": "税番号",
-				"taxIdPlaceholder": "例：PL123456789"
+				"taxIdPlaceholder": "例：T1234567890123"
```
Comment (hqod) [inline messages/jp.json:147]: Considering as Japanese translation, I use the Japanese tax number format as an example.

**Comment by zaiste** [comment]
@hqod thanks for this contribution and @RedstonePfalz for reviewing the translations!

**Comment by zaiste** [comment]
@hqod it's live! 🔥 https://demo-jp.yournextstore.com

**Pull request merged**